### ISSUE TITLE
TempHumBaro doesn't fit

### ISSUE THREAD

**Issue by Sateetje** [wez3/domoboard#55]
The text of a TempHumBaro sensor doesn't fit in the cell.

Full text is: 1.6C, 98 %, 1030 hPa

![temphumbar](https://cloud.githubusercontent.com/assets/16215404/21745039/5d613608-d522-11e6-905c-7487ff31e07a.png)

**Comment by Sateetje** [comment]
Moved this from #59 to this issue:

Domoticz support multiple kind of temperature sensors, like Temp, TempHum and TempHumBaro.

It would be nice when all of the types are supported including the Units (C, % and hPa).
"Type" : "Temp": "Data" : "23.0 C",
"Type" : "Temp + Humidity": "Data" : "5.6 C, 79 %",
"Type" : "Temp + Humidity + Baro": "Data" : "5.4 C, 98 %, 1031 hPa",

I've tried this solution, but this doesn't work (two or more times the same sensor) and the Units are lost.
Buiten = 357, thermometer-half, Temp
Vochtigheid = 357, thermometer-half, Humidity
Luchtdruk = 357, thermometer-half, Barometer
